lib/MetaCPAN/Script/Release.pm

```diff
@@ -490,6 +492,25 @@ sub import_tarball {
     }
 }
 
+sub set_changes_file {
```

Comment:
You'll just need to add some logic here to deal with `perl` distributions, since the Changes file there is `perldelta.pod`.  Have a look at the Changes controller.  Basically, once this work is complete and everything is reindexed, we'll want to use this new field rather than all of the logic in that controller.

So basically, you're taking the controller logic and applying it to the `changes_file` field so that we no longer have to discover where this file is on every request to the Changes controller.